web-client/src/ducks/myRequests/reducer.ts

```diff
@@ -1,9 +1,8 @@
-import { Post } from 'src/models/Post';
+import { Post } from 'src/models/posts';
```

Comment:
Changed to use Post model which will be shared between requests and offers and removed Offers specfic code.